mscr-ui/src/modules/crosswalk-editor/tabs/node-mappings/index.tsx

```diff
@@ -100,40 +95,43 @@ export default function NodeMappings(props: {
       uri: props.selectedCrosswalk.source.uri
     });
     mappings.predicate = predicateValue ? predicateValue : '0';
+    mappings.notes = notesValue;
     return mappings;
   }
 
   function closeModal() {
+    setNotesValue('');
     props.performMappingsModalAction('closeModal', null, null);
   }
 
-  function setSourceFilterValue(value: any) {}
+  function setSourceFilterValue(value: any) {
```

Comment:
Would you like to add a TODO or something, if this will be relevant in the future?